### PR TITLE
Check node status as part of validate-cluster.sh.

### PR DESCRIPTION
**Pull request by fabioy**
This should fix issue #7920. Checked the script on both Linux and Mac OS X (Yosemite).

### CODE REVIEW

**Comment by fabioy** [comment]
cc @roberthbailey

File: cluster/validate-cluster.sh [hunk 1]
```diff
@@ -30,13 +30,23 @@ trap 'rm -rf "${MINIONS_FILE}"' EXIT
...
+  #
+  #     NAME                     LABELS    STATUS
+  #     kubernetes-minion-03nb   <none>    Ready
+  #
+  # Echo the output, strip the first line, then gather 2 counts:
+  #  - Total number of nodes.
+  #  - Number of "ready" nodes.
+  "${KUBE_ROOT}/cluster/kubectl.sh" get nodes > "${MINIONS_FILE}" || true
+  found=$(cat "${MINIONS_FILE}" | sed '1d' | grep -c .) || true
+  good=$(cat "${MINIONS_FILE}" | sed '1d' | awk '{print $NF}' | grep -c '^Ready') || true
```
Comment (roberthbailey) [inline cluster/validate-cluster.sh]: nit: s/good/ready

Comment (fabioy) [inline cluster/validate-cluster.sh]: Renamed.

**Comment by roberthbailey** [comment]
LGTM.

**Comment by roberthbailey** [comment]
No reason to wait for travis/shippable because this is just changing a shell script. It'll be tested by jenkins e2e.

**Pull request merged**